axelor-account/src/main/java/com/axelor/apps/account/web/AnalyticDistributionTemplateController.java

```diff
@@ -35,14 +34,10 @@ public void validateTemplatePercentages(ActionRequest request, ActionResponse re
     try {
       AnalyticDistributionTemplate analyticDistributionTemplate =
           request.getContext().asType(AnalyticDistributionTemplate.class);
-      if (!Beans.get(AnalyticDistributionTemplateService.class)
-          .validateTemplatePercentages(analyticDistributionTemplate)) {
-        response.setError(
-            I18n.get(
-                "The configured distribution is incorrect, the sum of percentages for each axis must be equal to 100%"));
-      }
+      Beans.get(AnalyticDistributionTemplateService.class)
+          .validateTemplatePercentages(analyticDistributionTemplate);
```

Comment:
Please keep `response.setError` as catching the exception doesn't prevent further actions in `action-group` to be executed